omnigibson/action_primitives/starter_semantic_action_primitives.py

```diff
@@ -1761,7 +1767,7 @@ def _sample_pose_near_object(self, obj, pose_on_obj=None, **kwargs):
                 distance_lo, distance_hi = 0.0, 5.0
                 distance = (th.rand(1) * (distance_hi - distance_lo) + distance_lo).item()
                 yaw_lo, yaw_hi = -math.pi, math.pi
-                yaw = (th.rand(1) * (yaw_hi - yaw_lo) + yaw_lo).item()
+                yaw = th.tensor((th.rand(1) * (yaw_hi - yaw_lo) + yaw_lo).item(), dtype=th.float32)
```

Comment:
Do we really need th.tensor here? I dont think so.

Reply:
th.cos requires th.tensor input

Reply:
dont do .item()

Reply:
and remove the th.tensor here.